uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -709,6 +719,8 @@ define([
             }
 
             self.postContactSelection = false;
+
+            self._calculateOpeningAndClosingBalance();
```

Comment:
The order looks good to me I think, however, your question triggered a thought that we want to use the whole calculateBalanceChange() and not just _calculateOpeningAndClosingBalance(). This is safer as the balance.change may also vary depending on a contact.